tests/conf/chesapeake_cvpr_7.yaml

```diff
@@ -2,7 +2,7 @@ experiment:
   task: "chesapeake_cvpr"
   module:
     loss: "ce"
-    segmentation_model: "unet"
+    model: "unet"
     encoder_name: "resnet18"
     encoder_weights: null
     encoder_output_stride: 16
```

Comment:
```suggestion
    backbone: "resnet18"
    weights: null
    backbone_output_stride: 16
```

Reply:
I did not change the "encoder" naming here, because this file is not used by the `test_segmentation` trainer pytest. It is used by the Byol task so I thought I would change them separate, because otherwise byol task tests would fail.

Reply:
Thanks for the clarification. I'll let you update these in the BYOL PR then.